c7n/resources/chimesdkvoice.py

```diff
@@ -0,0 +1,37 @@
+from c7n.manager import resources
+from c7n.query import (
+    QueryResourceManager,
+    TypeInfo,
+    DescribeSource,
+    ConfigSource,
+)
+from c7n.tags import universal_augment
+
+
+class DescribeVoiceConnector(DescribeSource):
+    # override default describe augment to get tags
+    def augment(self, resources):
+        tagged_resources = universal_augment(self.manager, resources)
+        return tagged_resources
+
+
+@resources.register('chimesdkvoice-voiceconnector')
```

Comment:
regarding resource name, `chime-voice-connector` .. sdk is generic in this context., and voice doesn't need a repeat.

Reply:
Hi, not sure what you are proposing.
Can you clarify thanks.

---

The service api in boto is "chime-sdk-voice", so that seems a sensible prefix for this collection of resources. (The deprecated legacy API is "chime" but that's not the subject of this pr as far as I am aware).

Amongst the various resources Inside that API we have resources named such as "phone number", "sip rules" as well as "voice profile" and "voice connector", so "voice connector" seems a logical and obvious name in this case.

Reply:
Whilst the boto  API has "chime sdk voice" and "chime sdk meeting" the Aws console seems to have just "chime sdk" and it only has the voice resources.

The "chime sdk meetings" namespace and client is only accessible via the APIs .

So I dont know if all the different chime APIs and resources should be under merely "chimesdk" in the cloud custodian namespaces of just make cc names map directly to Aws API names as this extension appears to be doing?